crates/nu-command/src/commands/nu/command.rs

```diff
@@ -35,7 +40,6 @@ impl WholeStreamCommand for Command {
                 None,
             )
             .optional("script", SyntaxShape::FilePath, "script to run")
-            .rest(SyntaxShape::String, "...")
```

Comment:
These are needed for cases where you need to run the test binaries. 

`nu --testbin cococo arg1 arg2 argN`

`arg1`, `arg2`, and `arg3` are the rest arguments  and those get "passed in" to the `cococo` testbin. (It's not doing that, right now, actually.. but it's needed so that it can be valid to call `nu` like that.)

Reply:
Also needed when doing, say, `nu script1.nu script2.nu script3.nu`

Reply:
@andrasio hmmm - for the first case, I wonder if there's a better way to do it.

for the 2nd case do we support `nu script1.nu script2.nu`? I tried it and only the first one ran

Reply:
Investigating...

Reply:
@jonathandturner  For the second case, it seems that support was removed. I will bring it back though.

Reply:
sounds good, let me close this one